setup

```diff
@@ -146,4 +146,15 @@ for HOST_IP in "${NC_HOST_IPS[@]}" ; do
     NC_TRUSTED_DOMAIN_NO=$(($NC_TRUSTED_DOMAIN_NO+1))
 done
 
+# env var is set from the dockerfile
+if [ "$NC_IS_PATCHED" = true ]; then
+    $OCC config:system:set integrity.check.disabled --value="true" --type=boolean
+    # (un)comment and adjust following line depending on the use case,
+    # otherwise a warning is shown, still
+    $OCC integrity:check-app user_ldap
+
+    # integrity checks are done once on upgrade case, thus we can directly remove the flag again
+    $OCC config:system:delete integrity.check.disabled
```

Comment:
@LukasReschke fyi, almost as discussed, only i remove the flag immediately after rescanning.

Reply:
👍